doc/releases/changelog-0.37.0.md

```diff
@@ -6,17 +6,79 @@
 
 <h4>Execute faster with Default Tensor 🔗</h4>
 
-* The `default.tensor` device now supports the `tn` method to simulate quantum circuits using exact tensor networks.
+* A new `default.tensor` device is now available for performing
+  [tensor network](https://en.wikipedia.org/wiki/Tensor_network) and
+  [matrix product state](https://en.wikipedia.org/wiki/Matrix_product_state) simulations using the
```

Comment:
> performing tensor network and matrix product state simulations

I think this sounds like making tensor networks and contracting them generally, not in the context of circuits.
We could say something more like "... is now available for simulating quantum circuits (faster?) via tensor networks and matrix product states..."

Reply:
Thanks, fixed!